source/code/include/playfab/PlayFabPlatformMacros.h

```diff
@@ -29,4 +29,24 @@
 // Hence to specify only Windows, we have check for ! _DURANGO.
 #if defined(_WIN32) && !defined(_DURANGO)
 #define PLAYFAB_PLATFORM_WINDOWS
-#endif //_WIN32
\ No newline at end of file
+#endif //_WIN32
+
```

Comment:
This section is very different from the rest of the file, and the purpose stated in the file's header comment block. Consider creating a new PlayFabPlatformTypes.h

Reply:
If I get a second vote agreeing with you, I'll do that.

Reply:
I agree with jeremy

Reply:
Done.